app/services/analytics_events.rb

```diff
@@ -4030,14 +4029,14 @@ def user_registration_phone_setup_visit(enabled_mfa_methods_count:, **extra)
   end
 
   # Tracks when user's piv cac is disabled
-  def user_registration_piv_cac_disabled
-    track_event('User Registration: piv cac disabled')
+  def piv_cac_disabled
+    track_event('Piv cac disabled')
```

Comment:
the casing here is inconsistent with below, can we make it match?
```suggestion
    track_event('Piv Cac disabled')
```